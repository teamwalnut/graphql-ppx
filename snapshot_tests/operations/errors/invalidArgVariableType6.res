/**
 * Lists should be generic and not their innermost type, so this should fail
 */
module MyQuery = %graphql(`
  query invalidArgValues($arg: String) {
    invalidArgValues(
        list: $arg
        stringRequired: ""
    )
  }
`)
