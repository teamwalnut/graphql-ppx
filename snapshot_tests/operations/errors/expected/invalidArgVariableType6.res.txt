
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/invalidArgVariableType6.res[0m:[2m7:15-18[0m

  5 [2mâ”†[0m query invalidArgValues($arg: String) {
  6 [2mâ”†[0m   invalidArgValues(
  [1;31m7[0m [2mâ”†[0m       list: [1;31m$arg[0m
  8 [2mâ”†[0m       stringRequired: ""
  9 [2mâ”†[0m   )
 10 [2mâ”†[0m }

  Invalid argument. Expected "[String]" but received "String".

